webdriver/tests/permissions/set.py

```diff
@@ -54,11 +54,6 @@ def test_non_secure_context(session, url, state):
     assert_error(response, "invalid argument")
 
 @pytest.mark.parametrize("state", ["granted", "denied", "prompt"])
-@pytest.mark.parametrize("realmSetting", [
```

Comment:
I'm not familiar with this part of the code base, but I wonder if any of the tests below can be simplified/removed now that there's no realm handling?

Reply:
I don't think so... none of them did anything useful in practice as no one actually implemented this.

Reply:
OK. The test failures being reported here look real though; I think you also need to remove `realmSettings` from the test functions for them to pass again.

Reply:
Ok, thanks for checking that. Will do!